### ISSUE TITLE
Multiple key for same value

### ISSUE THREAD

**Issue by timoteo7** [vue-generators/vue-form-generator#634]
* **I'm submitting a ...**
  * [X] feature request

* **What is the current behavior?**

```
{
  "fields": [
  {
    "type": "input",

    "title": "Name",
    "label": "Name",
    "placeholder": "Name",

    "htmlType": "text",
    "inputType": "text",

    "name": "name",
    "model": "name",
  },
 ]
}
```

- **What is the motivation / use case for changing the behavior?**

use the same JSON to more than one components
(some special character to separate multiple keys)

```
{
  "fields": [
  {
    "type": "input",

    "title,label,placeholder": "Name",

    "htmlType,inputType": "text",

    "name,model": "name",
  },
 ]
}
```

* **Please tell us about your environment:**

  * Version: [2.3.4]
  * Browser: [Firefox 68 ]
  * Language: [ES6/7]

**Comment by zoul0813** [comment]
This request complicates the schema and would create a confusing document that’s hard to parse.

It’s also not supported by any of the underlying libraries we use for setting object properties (lodash).

Sorry, but I’m going to close this request with a “won’t implement”.

**Comment by zoul0813** [comment]
You can very easily implement a schema generator that works with your custom schema recommendation, that generates a valid VFG schema though.  If you do, feel free to let us know and we’d be more than happy to link to it as a third party plugin.
